frontend/src/components/Menu/Sidebar.vue

```diff
@@ -28,10 +28,14 @@
             <b-icon icon="layers" aria-hidden="true"></b-icon>
             <span class="ml-2">{{ $t('GDT') }}</span>
           </b-nav-item>
-          <b-nav-item to="/information" active-class="activeRoute">
+          <b-nav-item to="/information" class="mb-3" active-class="activeRoute">
             <b-img src="/img/svg/info.svg" height="20" class="svg-icon" />
             <span class="ml-2">{{ $t('navigation.info') }}</span>
           </b-nav-item>
+          <b-nav-item to="/usersearch" active-class="activeRoute">
+            <b-img src="/img/svg/loupe.png" height="20" class="svg-icon" />
+            <span class="ml-2">{{ $t('navigation.usersearch') }}</span>
+          </b-nav-item>
```

Comment:
it is a png icon, so please don't use svg-icon class, maybe this is the cause for the black symbol, if choosen.

Reply:
Danke für deine wertvollen Anmerkungen...